### PR TITLE
usage Tracker and whitelist

### PR DESCRIPTION
**Pull request by Eyobyb**
this is the updated usage tracker with better folder structure

### CODE REVIEW

File: apps/slackbot/utils.py [hunk 1]
```diff
@@ -110,8 +112,10 @@ def count_string_tokens(string: str, model_name: str) -> int:
     return len(encoding.encode(string))
 
 
-def chunk_and_summarize(text_data: str, question: str, open_ai_key: str, link: str):
-    llm = OpenAI(temperature=0.9, openai_api_key=open_ai_key)
+def chunk_and_summarize(text_data: str,  question: str, open_ai_key: str, link: str, team_id: str = None, user_id: str = None):
+    llm = SherpaOpenAI(
+        temperature=0.9, openai_api_key=open_ai_key, user_id=user_id, team_id=team_id)
```
Comment (20001LastOrder) [inline apps/slackbot/utils.py:117]: Here we should use the configured temperature

**Review by 20001LastOrder — CHANGES_REQUESTED** [review]
Looks good except one small change

File: apps/slackbot/utils.py (hunk 1 above)
Comment (Eyobyb) [reply apps/slackbot/utils.py:117]: fixed it check it out

Comment (Eyobyb) [reply apps/slackbot/utils.py:116]: I forgot to change it inside the utility that was the case

Comment (Eyobyb) [reply apps/slackbot/utils.py]: its changed now

Comment (Eyobyb) [reply apps/slackbot/utils.py]: its changed now @20001LastOrder

**Review by 20001LastOrder — APPROVED** [review]

**Pull request merged**